pnpm-lock.yaml

```diff
@@ -12436,10 +12436,6 @@ packages:
     resolution: {integrity: sha512-yqcRVnjf+qS+tC4NxOKLJOaSJ+csHmh/dHUzvCTkf5rLsplwXYRnny2r0tqGTQ4tuXMxwgSMKPYwicg81P+xuw==}
     engines: {node: '>=12'}
 
-  '@sentry-internal/feedback@7.109.0':
```

Comment:
Why are there so many changes not related to storyly in the pnpm-lock file ?

Reply:
i'll figure out, i've done alot of llm ci related deps changes on my local branch , i don't know if it's related

Reply:
Well it seems to be related to the bump of storyly version

Reply:
it's dublicated packages in pnpm-lock.yaml